src/stories/table.stories.js

```diff
@@ -341,3 +341,54 @@ export const Responsive = () => html`
     </cod-table-body>
   </cod-table>
 `;
+
+export const Stacked = () => html`
+  <cod-table data-id="table-stacked" data-stacked="true">
+    <cod-table-header>
+      <cod-table-row>
+        <cod-table-cell-header scope="col"
+          >Bus Route Number</cod-table-cell-header
+        >
+        <cod-table-cell-header scope="col"
+          >Bus Route Name</cod-table-cell-header
+        >
+        <cod-table-cell-header scope="col">Route Viewer</cod-table-cell-header>
+        <cod-table-cell-header scope="col"
+          >Downloadable Schedule</cod-table-cell-header
+        >
+      </cod-table-row>
+    </cod-table-header>
+    <cod-table-body>
+      <cod-table-row>
+        <cod-table-cell data-label="Bus Route Number">1</cod-table-cell>
+        <cod-table-cell data-label="Bus Route Name">Vernor</cod-table-cell>
+        <cod-table-cell data-label="Route Viewer">
+          <a href="https://example.com">View Route</a>
+        </cod-table-cell>
+        <cod-table-cell data-label="Downloadable Schedule">
+          <a href="https://example.com">Download Schedule</a>
+        </cod-table-cell>
+      </cod-table-row>
+      <cod-table-row>
+        <cod-table-cell data-label="Bus Route Number">2</cod-table-cell>
+        <cod-table-cell data-label="Bus Route Name">Michigan</cod-table-cell>
+        <cod-table-cell data-label="Route Viewer">
+          <a href="https://example.com">View Route</a>
+        </cod-table-cell>
+        <cod-table-cell data-label="Downloadable Schedule">
+          <a href="https://example.com">Download Schedule</a>
+        </cod-table-cell>
+      </cod-table-row>
+      <cod-table-row>
+        <cod-table-cell data-label="Bus Route Number">3</cod-table-cell>
+        <cod-table-cell data-label="Bus Route Name">Grand River</cod-table-cell>
+        <cod-table-cell data-label="Route Viewer">
+          <a href="https://example.com">View Route</a>
+        </cod-table-cell>
+        <cod-table-cell data-label="Downloadable Schedule">
+          <a href="https://example.com">Download Schedule</a>
+        </cod-table-cell>
+      </cod-table-row>
+    </cod-table-body>
+  </cod-table>
```

Comment:
@sreidthomas these are the HTML tags that we'll use for the bus schedule view table once this PR gets merged in.